testsuite/single_node_performance.py

```diff
@@ -46,6 +47,7 @@ class Flow(Flag):
 
 MAX_BLOCK_SIZE = int(os.environ.get("MAX_BLOCK_SIZE", default=DEFAULT_MAX_BLOCK_SIZE))
 NUM_BLOCKS = int(os.environ.get("NUM_BLOCKS_PER_TEST", default=15))
+NUM_BLOCKS = 50
```

Comment:
We set NUM_BLOCKS and then immediately override it with 50?

Reply:
I've configured CI to run once as if mainnet validation, to see numbers to update, and will then revert these

Reply:
updated and removed these